src/defaults.js

```diff
@@ -1433,7 +1433,7 @@ var defaults = {
      * @default
      * @memberOf module:defaults
      */
-    columnGrabMargin: 5,
+    columnGrabMargin: 10,
```

Comment:
This makes it a little easier to drag columns to move them or remove them.
We should change the behavior completely though going forward. This is required right now as both the ColumnSelection and ColumnMoving features depend on the mouse down event - this grab margin effectively splits the header cell into two regions, the top pixels are where the column is grabbed, the bottom is where its selected.

A more sensible approach would be for the ColumnSelection feature only select on the mouse up event (instead of down), and to not select if up event is on a different cell than the down event. That way both the ColumnSelection and ColumnMoving could operate on the whole column header.